app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/vaultunlock/VaultUnlockViewModel.kt

```diff
@@ -208,6 +208,9 @@ class VaultUnlockViewModel @Inject constructor(
 
     private fun handleUnlockClick() {
         val activeUserId = authRepository.activeUserId ?: return
+
+        if (state.input.isEmpty()) return
```

Comment:
Display a dialog indicating the input field is required.

Reply:
Done, but I don't know how you work with translation, so I just add raw strings now. 
If you have some process over strings in project, let me know what I have to do.

Reply:
I think, it isn't best UX on error handle. But I have no idea how to make it better in general.